app/controllers/comments_controller.rb

```diff
@@ -1,56 +1,55 @@
 class CommentsController < ApplicationController
   before_action :authenticate_user!, only: [:new, :edit, :create, :update, :destroy]
     def index
```

Comment:
za daleko jedno wcięcie wystarczy

To jest na równi z `before_action ` (jak i reszta)